app/jobs/lifetime_usages/recalculate_and_check_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module LifetimeUsages
+  class RecalculateAndCheckJob < ApplicationJob
+    unique :until_executed, on_conflict: :log
```

Comment:
Should we use the `default` queue. I was thinking at using the `billing` one, so that is does not take precedence over `mailers` and `provider`, what do you think?

Reply:
I had not considered that, I'll move it to the `billing` queue 👍